### ISSUE TITLE
Do we need to update the projects section?

### ISSUE THREAD

**Issue by williammartin** [openservicebrokerapi/servicebroker#564]
It looks like the project section has not been updated looking forward to 2.15? https://github.com/openservicebrokerapi/servicebroker/projects/1

Should this be added to a list of instructions on release, or is it something that will be done during a call?

**Comment by mattmcneeney** [comment]
Yes - updating the roadmap should be added to the contributing guide when cutting a release. I'll take this one.
